.wordlist.txt

```diff
@@ -245,9 +245,6 @@ dibc
 disjuncts
 DOM
 EcmaScript
-encourageMe
-encouragementBot
-encouragementinstallation
 enduser
```

Comment:
Just to be sure, should "installation" be capitalized there like the other post-encouragment words?

Reply:
I'm not sure I understand. These words are no longer in the documentation repo, so I'm taking them out.

Reply:
Oh, duh. Since there was only red, I misinterpreted it as adding for some reason. Never mind.